requirements-full.txt

```diff
@@ -14,8 +14,8 @@ psutil==5.8.0
 lxml==4.6.5
 opencv-python==4.5.1.48
 spikeextractors==0.9.9
-spikeinterface==0.93.0
-neo==0.10.0
+spikeinterface @ git+https://github.com/SpikeInterface/spikeinterface.git
+neo>0.10.0
```

Comment:
`requirements-full.txt` uses exact versioning; I'd return this to `==` and pin it to the exact version needed, such as `0.10.1`.

If you need to, update minimal requirements in `requirements-minima.txt` with lower bound versioning as `>=` the precise minimal version needed.

Reply:
Done.